app/containers/HomeContainer/index.js

```diff
@@ -103,6 +123,37 @@ export function HomeContainer({
       )
     );
   };
+
+  const customRenderRepoList = () => {
+    const items = _.get(reposData, 'items', []);
+    const totalCount = _.get(reposData, 'totalCount', 0);
+    return (
+      ((flag && items.length !== 0) || loading) && (
+        <CustomCard>
+          <Skeleton loading={loading} active>
+            {repoName && (
+              <div>
+                <T id="search_query" values={{ repoName }} />
+              </div>
+            )}
+            {totalCount !== 0 && (
+              <div>
+                <T id="matching_repos" values={{ totalCount }} />
+              </div>
+            )}
+
+            {totalCount >= 1 && (
```

Comment:
we should be resuing the same code for both the cases. 

feeling lucky / full search. 

the action that is dispatched in both cases should be different so that the saga can handle the logic of what to return.

Reply:
What dose problem I don't understood